### ISSUE TITLE
Support 0.182 multiple project

### ISSUE THREAD

**Issue by grenierdev** [icetee/remote-ftp#44]
- Seperate Client and Tree View : 1x Tree View for the editor, Nx Client per root folder
- `atom.project.onDidChangePaths` to get noticed when a root folder changes

**Comment by grenierdev** [comment]
- `atom.project.getPaths()` -> `atom.project.getDirectories()`
- `atom.project.resolve` no longer work, should use `Directory.resolve` instead

**Comment by GStarSade** [comment]
Thank, I would really like this feature :) :+1: 
But tell me has this been implemented in the latest update? If so I could use more clearer instructions. Thanks.

**Comment by Yehonal** [comment]
waiting for this feature ! it's really useful

**Comment by tlygnersjo** [comment]
Any news about this planned feature?

**Comment by kikoseijo** [comment]
Hi there, great job, love the plugin.
Bye bye DW, Welcome ATOM. And you guys, without you could not be posible.

10 points for this feature,
What about choosing the Project Folder to work with, no need to do it automatic for now, just have a select with the open project folder list could be posible? 

This way you can just change the path to read before connections.
If this helps!! ;-)

GOOD job again.

**Comment by ol1s** [comment]
At the moment I am just opening separate ATOM windows for each project I want to work on - but is a bit clunky

**Comment by grenierdev** [comment]
Just to let you guys know that I've looked into something an engineer at Facebook told me last year concerning this feature. It seems like Atom now support the principle of virtual filesystem through a _directory provider_. Turns out this is pretty simple to implement.

Unfortunately, Atom's tree-view do not support this feature and is hard coded to the file system.

[Commented on an existing issue on the tree-view project](https://github.com/atom/tree-view/issues/452#issuecomment-225460026).
